### ISSUE TITLE
Change Timezone

### ISSUE THREAD

**Issue by mbpictures** [Innologica/vue2-daterange-picker#214]
Hi!
First of all: Thanks for your awesome work!
But I have a question: is it possible to change the displayed time to GMT-Time? It seams like its always user time.

Thanks for your help and best wishes!

**Comment by nkostadinov** [comment]
Well the components uses Datetime objects, you can easily display whatever timezone you want using slots - https://innologica.github.io/vue2-daterange-picker/advanced/#slots-demo

**Comment by nick-falco** [comment]
Sorry to comment on an old thread, but it seems most applicable to ask here.

The slots approach works if you just want to display an arbitrarily formatted date. Alternatively I've found that it's possible to specify the date format in the locale-data property.

e.g.

```
dateLocale: {
    format: 'UTC:yyyy-mm-dd\'T\'HH:MM:ss\'Z\'',
},
```

The issue with this solution is that the "time picker" is always in local time. In my case I want to allow for the selection of UTC dates, and I am currently in US Central Standard Time (CST). In the image below I've included an example where the hour and minute are both set to zero. In this image you'll see that since specifying a "format" or using slots only changes the displayed date to UTC, selecting a time of hour "00" minute "00" results in hour "06" minute "00" in the outputted UTC date. This makes for a really weird user interface where the user would have to compute the time zone calculation themselves. 

![image](https://user-images.githubusercontent.com/20054276/107311807-a7350900-6a54-11eb-98f9-9ba0d06112d1.png)

To solve this problem, it would be great if the actual date / time picker was time zone aware, rather than just changing the display format.
